helm-plugins/snapshot/plugin.yaml

```diff
@@ -0,0 +1,14 @@
+name: "snapshot"
+version: "0.1.0"
+usage: "Create a visual snapshot of your packaged Helm chart as a Meshery Snapshot"
+description: "A Helm plugin to generate Meshery Snapshots from Helm charts using Go"
+command: "$HELM_PLUGIN_DIR/helm-plugin-snapshot"
+hooks:
+  install: |
+    echo 'Installing Meshery Snapshot Plugin...'
+    # No need to manually install here since Helm manages this.
```

Comment:
@Aijeyomah How does this work? Shouldn't we need to download the binary (the go program) and place it in the "HELM_PLUGIN_DIR/bin"?